source/_components/abode.markdown

```diff
@@ -159,10 +137,10 @@ This integration will also list all Abode `Quick Actions` that are set up. You c
 
 ### Light
```

Comment:
Maybe we should just remove these 2 sections? 🤷‍♂

Reply:
Agreed. I actually removed the bottom three sections. The light and switch section were just repeat of information above. Same with the binary sensor although it had an extra sentence which I just moved above.